src/pages/magic/deck/index.tsx

```diff
@@ -354,6 +371,7 @@ const CardComponent: React.FunctionComponent<CardComponentProps> = ({ name, imag
                                 <img className={ styles.tooltipCard } src={ tooltip.backFace.image_uris.border_crop } alt={ tooltip.backFace.name } />
                             </div>
                             <Tooltip id={ tooltip.backFace.name } { ...tooltip.backFace } />
+                            { tooltip.all_parts && filterAllParts(tooltip).map((_, i) => <div key={ i } className={ styles.cardSpacer } />) }
```

Comment:
fixes alignment when the front face has tokens alongside it and the back face does not